sdk/core/create-account.mdx

```diff
@@ -53,3 +92,61 @@ const transaction = await rhinestoneAccount.sendTransaction({
 })
 console.log('Transaction', transaction)
 ```
+
+## Embedded Wallet Integration
+
+For production applications, choose the appropriate signer type:
+
+- **New users**: Use embedded wallets like [Dynamic](./signers/dynamic) or [Privy](./signers/privy)
+- **Existing crypto users**: Support [external wallets](./signers/external-wallet) 
+- **High security**: Implement [passkeys](./passkeys) for biometric authentication
+- **Enterprise**: Use [Turnkey](./signers/turnkey) for robust key management
+
+## Advanced Configuration
+
+### Account Providers
+
+Rhinestone supports multiple ERC-7579 compatible smart account implementations:
+
+```ts
+const account = await createRhinestoneAccount({
+  owners: { type: 'ecdsa', accounts: [signer] },
+  accountProvider: 'nexus', // 'safe', 'kernel', 'startale', 'prime'
+  rhinestoneApiKey,
+})
+```
+
+**Learn more**: [Smart Account Providers](../customize/smart-account-providers/nexus)
+
+### Multi-Signature Accounts
+
+Create accounts requiring multiple signatures:
+
+```ts
+const multiSigAccount = await createRhinestoneAccount({
+  owners: {
+    type: 'multisig',
```

Comment:
Should be `type: 'ecdsa'`?

Reply:
yeah sorry my bad updated it, can be `ecdsa, passkey, multi-factor`